### ISSUE TITLE
Trouble running in CoLab

### ISSUE THREAD

**Issue by jimmybuffi** [man-group/dtale#787]
I tried both options for running in CoLab...

With USE_COLAB = True, I got a link to message that simply said there was an error.

With USE_NGROK = True, and after installing ngrok and adding my auth token, i still got the link to this page:

http://1b2d-34-125-95-50.ngrok.io/dtale/main/9

Is there a specific mechanism by which ngrok needs to be started before dtale will run?

Any advice regarding why either might be failing? Thank you!

**Comment by wamaw123** [comment]
I am having a similar issue. D-Tale worked fine last week and now any link on colab leading to error-404. It seems the method used to keep the cell alive is not working anymore. Maybe something on colab side ? Ngrok does not work either.

**Comment by aschonfeld** [comment]
So I just went into Colab anmd everything worked fine for me: https://colab.research.google.com/drive/1ZztkdLQTawVu732W2n1kagy9BmfO2Vm-?usp=sharing

In particular I ran `!pip install -U dtale` and then the following:
```
import pandas as pd
import dtale
import dtale.app as dtale_app

dtale_app.USE_COLAB = True
dtale.show(pd.DataFrame(dict(x=[1,2,3], y=[2,4,8]))
```

Then clicked the resulting link. Hope this helps

**Comment by angelina-akin** [comment]
it worked in early december but no longer works for me as well. Ran my old notebook and now getting an error.

**Comment by aschonfeld** [comment]
@angelina-akin I've figured out the issue with colab.  I will have this updated in the next release but for now here's a workaround:
```
import pandas as pd
import dtale
import dtale.app as dtale_app

dtale_app.USE_COLAB = True
dtale_app.ACTIVE_HOST = '0.0.0.0'
dtale.show(pd.DataFrame(dict(x=[1,2,3], y=[2,4,8]))
```

Let me know if that works. Thanks

**Comment by aschonfeld** [comment]
@angelina-akin  just released v3.10.0 to pypi (should be on conda-forge soon) with the colab update included (you will no longer have to include `host='0.0.0.0'`).

Also, if you haven't already, please put your ⭐ on the repo when you get a sec. Thanks! 🙏

**Comment by angelina-akin** [comment]
@aschonfeld okay will give it a try now. Thanks!
